tpm2_pytss/ESAPI.py

```diff
@@ -976,11 +984,20 @@ def Certify(
         signHandle,
         qualifyingData,
         inScheme,
-        session1=ESYS_TR.NONE,
-        session2=ESYS_TR.NONE,
+        session1=ESYS_TR.PASSWORD,
+        session2=ESYS_TR.PASSWORD,
         session3=ESYS_TR.NONE,
     ):
 
+        check_handle_type(objectHandle, "objectHandle")
```

Comment:
@whooo just a few thoughts before I head out on a few days of vacation (back May 3):
  - We probably want to type check the handles, so I added this routine.
  - The type check of other data, the get_cdata routine you added, we should probably pass the variable name so TypeErrors say explicitly that this argument failed. This way users get explicit feedback that should help avoid ambiguity.

Thanks again for your help along the way, much appreciated.